### PR TITLE
drivers: doxygen cleanup

### PR DESCRIPTION
**Pull request by aabadie**
I went through the drivers documentation and fixed a few things:
* ensure all doxygen groups are called `drivers_<name>`
* fixed `@fixed` where `@name` should be used
* other things that I don't remember

### CODE REVIEW

File: drivers/cc110x/include/cc110x-internal.h [hunk 1]
```diff
@@ -206,5 +206,5 @@ typedef struct cc110x_statistic {
 }
 #endif
 
-/** @} */
 #endif /* CC110X_INTERNAL_H */
+/** @} */
```
Comment (haukepetersen) [inline drivers/cc110x/include/cc110x-internal.h]: here and below: newlines at EOF missing

File: drivers/hd44780/include/hd44780_internal.h [hunk 2]
```diff
@@ -24,7 +24,7 @@ extern "C" {
 #endif
 
 /**
- * @brief   HD44780 LCD commands
+ * @name   HD44780 LCD commands
```
Comment (haukepetersen) [inline drivers/hd44780/include/hd44780_internal.h]: indention now off by 1 space (here and below)

File: drivers/include/at30tse75x.h [hunk 3]
```diff
@@ -122,46 +122,40 @@ typedef enum {
...
 /**
   * @brief   Polarity of the ALERT pin
-  * @{
   */
 typedef enum {
     AT30TSE75X_ALARM_ACTIVE_LOW,
     AT30TSE75X_ALARM_ACTIVE_HIGH
 } at30tse75x_alarm_polatity_t;
+
 /** @} */
```
Comment (haukepetersen) [inline drivers/include/at30tse75x.h]: It think you meant to remove this line instead of adding a new one?

File: drivers/include/diskio.h [hunk 4]
```diff
@@ -59,15 +59,13 @@ typedef enum {
 } diskio_result_t;
 
 /**
- * @name Disk Status Bits
- * @{
+ * @brief Disk Status Bits
```
Comment (haukepetersen) [inline drivers/include/diskio.h]: indention broken

File: drivers/include/dsp0401.h [hunk 5]
```diff
@@ -31,8 +31,7 @@ extern "C" {
 #endif
 
 /**
- * @name Return codes
- * @{
+ * @brief Return codes
```
Comment (haukepetersen) [inline drivers/include/dsp0401.h]: same

File: drivers/sx127x/include/sx127x_registers.h [hunk 6]
```diff
@@ -1424,3 +1424,4 @@ extern "C" {
 #endif
 
 #endif /* SX127X_REGISTERS_H */
+/** @} */
```
Comment (haukepetersen) [inline drivers/sx127x/include/sx127x_registers.h]: newline missing

File: drivers/include/at30tse75x.h (hunk 3 above)
Comment (aabadie) [reply drivers/include/at30tse75x.h]: In fact, this line closes the `@name` of line 104 above. I just wanted to make it more explicit. Will add a comment

**Comment by aabadie** [comment]
@haukepetersen, pushed an update for at86rf2xx and mrf24j40 drivers. There were remaining inconsistencies in the rendered doxygen. I'm not sure what to do with the tx power mapping in mrf24j40.

File: drivers/cc110x/include/cc110x-netdev.h [hunk 7]
```diff
@@ -64,3 +64,4 @@ int netdev_cc110x_setup(netdev_cc110x_t *netdev_cc110x, const cc110x_params_t *p
 #endif
 
 #endif /* CC110X_NETDEV_H */
+/** @} */
```
Comment (haukepetersen) [inline drivers/cc110x/include/cc110x-netdev.h]: newline at EOF

File: drivers/include/bmp180.h [hunk 8]
```diff
@@ -29,16 +29,14 @@ extern "C" {
 #endif
 
 /**
- * @name Oversampling modes
- * @{
+ * @brief Oversampling modes
```
Comment (haukepetersen) [inline drivers/include/bmp180.h]: indention

**Comment by aabadie** [comment]
@haukepetersen, I made another pass of cleanup (mostly indentation problems) and fixed 'problems' in `lis3dh.h` files. Should be better now.

**Comment by haukepetersen** [comment]
damn, needs a rebase, though...

**Comment by aabadie** [comment]
@haukepetersen, rebased

**Comment by aabadie** [comment]
@haukepetersen, is your ACK still valid ?

**Comment by haukepetersen** [comment]
yes, it is :-)

**Pull request merged**